composer.json

```diff
@@ -12,8 +12,9 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-json": "*",
         "ckr/arraymerger": "^2.0",
-        "hkarlstrom/openapi-reader": "^0.2.1",
+        "hkarlstrom/openapi-reader": "~0.2",
```

Comment:
To my understanding `~0.2` means>=0.2 and < 1.0.0. As the YAML support was added to the reader in 0.3, this should be `^0.3`?

Reply:
Yes, you're right: I wasn't considering the whole situation.

Reply:
Im not sure either, but I think you can just push to your branch and the pull request will be automatically updated. If it's not, just create a new one.

Reply:
Ummm, yep, I think it's automatic: I can see my commit already in the list of this pull request.